adapters/adform/adform_test.go

```diff
@@ -597,3 +599,81 @@ func TestPriceTypeUrlParameterCreation(t *testing.T) {
 		}
 	}
 }
+
+// Asserts that toOpenRtbBidResponse() creates a *adapters.BidderResponse with
+// the currency of the last valid []*adformBid element and the expected number of bids
+func TestToOpenRtbBidResponse(t *testing.T) {
+	// expected values
```

Comment:
The flow of this test is very clean. I recommend to remove all of the comments, I don't think they're necessary.

Reply:
Done